### ISSUE TITLE
Please import only the functions you need from data.table

### ISSUE THREAD

**Issue by tdhock** [psychbruce/PsychWordVec#1]
hi @psychbruce 

In PsychWordVec.R there is `@import data.table` in your roxygen comments, which get converted to `import(data.table)` in your NAMESPACE, which has the effect of importing everything that data.table exports.
In the new version of data.table on github master, `remotes::install_github("Rdatatable/data.table")` there is a new exported object `%notin%`.
There is also an exported object with that same name in package bruceR which you also import.
This has the effect of generating the following new warning when I run R CMD check PsychWordVec with data.table github master installed:
```
* checking whether package 'PsychWordVec' can be installed ... WARNING
Found the following significant warnings:
  Warning: replacing previous import 'data.table::%notin%' by 'bruceR::%notin%' when loading 'PsychWordVec'
```
The fix would be to use `@importFrom data.table data.table` etc (for only the objects you use), instead of `@import data.table`.
Please update your code, then run R CMD check PsychWordVec_version.tar.gz with data.table master installed.
After you update your code and R CMD check has no more errors, please submit a new version of PsychWordVec to CRAN.
This will help facilitate releasing a new version of data.table to CRAN.
(data.table devs must make sure all reverse dependencies do not break, before submitting a new version to CRAN)

**Comment by psychbruce** [comment]
Hi @tdhock , thanks for reporting this issue. Since it was caused by `%notin%`, I have now removed the import of `bruceR::%notin%` and would submit a new version of `PsychWordVec` to CRAN in the following days.

**Comment by tdhock** [comment]
thanks for the quick reponse!

**Comment by psychbruce** [comment]
@tdhock Issue fixed in `PsychWordVec` CRAN version 0.2.0

**Comment by tdhock** [comment]
thanks I confirm the warning has gone away on my end too.
